### DISCUSSION TITLE
Multiple default recipients

### DISCUSSION THREAD

**Discussion by stites** [anonaddy/anonaddy#334] [Feature Requests]
I would like to add multiple default recipients for my domain, but I see that this is not possible. Is there a technical limitation for this or would this be a valid feature request?

**Comment by willbrowningme** [comment]
There isn't a technical limitation, I would just have to change the database structure slightly for that relationship. How many would you like to have as default recipients?

**Reply by moi-virtual** [reply]
Two is what I would use now. At sneakemail.com, I used three, at one time, but now am at two.

**Comment by stites** [comment]
I would only need two!

**Comment by vorpalhex** [comment]
This would be a big boon for my case as well, since I want a domain where both my spouse and I will each get a copy of emails.

**Comment by http403** [comment]
This is quite dead. Any updates? I wish I can use it as kind of group mail for my family, even if that means pay more.

**Reply by http403** [reply]
Given a family is two adult plus 1 to 2 childs, with some leeway, I think 5 is suffient.

**Reply by http403** [reply]
I just saw the Roadmap and this was high on the list. It is quite long so I should be applogize for asking.

https://github.com/anonaddy/anonaddy/projects/1#card-68636917

**Reply by roke-julian-lockhart** [reply]
> https://github.com/anonaddy/anonaddy/discussions/334#discussioncomment-6663880

@http403, I know a family of 15. It needs to allow an arbitrary amount.

**Reply by http403** [reply]
But without limit may cause the Addy's mail server to overload and possiblity blacklisted?

**Reply by roke-julian-lockhart** [reply]
Despite that, @http403, there *will* be families who need more than whatever limit is defined, and if we just decide an arbitrary limit, that is rather discriminatory against them. The sole alternative option I can envision is verifying family members' identities using their legal documents, but that's much too much work for such a small business as this, surely?

**Reply by http403** [reply]
I took the median US household number and plus 1 to it. I would propose an arbitrary limit for the Anon service. If a family need more then that, they would need to self host it, or have a more expensive plan that have its dedicated IP.

**Reply by roke-julian-lockhart** [reply]
Yeah, a dedicated plan perhaps wouldn't be too bad, perhaps.

**Reply by http403** [reply]
I don't believe [willbrowningme](https://github.com/willbrowningme) can handle too much edge cases. The service is for personal use, and this request outgrew that. The multiple recipents feature I believe is for people who have multiple mailboxes to seprate from personal, work, or whatnot, not for family members.

**Reply by stites** [reply]
It looks like @willbrowningme is aiming for full ">1 support" (the description is "Allow users to add more than one default recipient for their custom domains and additional usernames."). Someone can file another issue if this final solution is finite and they need a longer list of recipients.

There is a lot of speculation in this thread -- it's probably best to just suspend your expectations and let the devs figure out impact on infrastructure, if this feature results in blacklisting, what the cost of infrastructure will be, whether the incurred cost will be pushed onto us users, what edge cases are, what the addy.io service will support.

**Comment by a-lam** [comment]
Rather than having this feature specific to a domain, can we make a rule? I would like to give the ability for my spouse to create new aliases that go to either them or both of us. Something like spouse.*@customdomain.com would be theirs couple.*@customdomain.com would be for both of us while remaining *@customdomain.com would be for myself.

**Reply by a-lam** [reply]
I meant to display spouse.wildcard@customdomain.com but it seems like this markup strips out the asterix.

**Reply by stites** [reply]
This seems like a new feature request (and you should file it as such!). I am only intending the fix for this issue to touch on the default recipients (so a type-change from `string` to `list string`).

**Reply by roke-julian-lockhart** [reply]
@a-lam, [`issues/512#issuecomment-2186795939`](https://github.com/anonaddy/anonaddy/issues/512#issuecomment-2186795939) implements this.

**Comment by moi-virtual** [comment]
I too would like more than one default recipient. I already have this ability in every other place where I setup aliases. In every other respect, I prefer to use anonaddy.
